packages/design-system/src/styles/components/_ChoiceList.scss

```diff
@@ -87,9 +86,9 @@ label,
   display: grid;
   grid-area: check;
   height: var(--choice__size);
-  justify-items: center;
   margin: 0;
   place-content: center;
+  place-items: center;
```

Comment:
Linter combined `justify-items` and `align-items` into shorthand property

Reply:
😮